### ISSUE TITLE
Do we need to expose the licensing info and version info over HTTP so Service Pulse can report on it?

### ISSUE THREAD

**Issue by gbiellem** [Particular/ServiceControl.Monitoring#16]
Do we need to expose the licensing info and version info over HTTP so Service Pulse can report on it?
This is something we do for the ServiceControl service in SP.

//cc @Particular/launchwave-devopsadvancedmonitoring-squad

**Comment by SzymonPobiega** [comment]
I think that depends if SP will be usable without a working connection to SC. I guess it should. But exposing the licensing info from both SC and SC.Monitoring will cause problems if these info sources are not consistent.

**Comment by gbiellem** [comment]
The scenario I was thinking about was if  we had the services split over two machines,  In that case the licensing would be from HKLM on each machine so it could be inconsistent and reporting on it would be a good thing.  Otherwise both should be read from the same HKLM location so they would be consistent

**Comment by SzymonPobiega** [comment]
@gbiellem makes sense. Can say for the whole squad by IMO (based on the scenario you described):
 * "do we need?" - no
 * "would it be good to?" - yes

**Comment by WilliamBZA** [comment]
Related issue: https://github.com/Particular/ServicePulse/issues/436
